app/static/js/page.js

```
@@ -77,269 +233,642 @@ function snakeOilApp() {
+        // Calculate "No Vendehumo" score as complement of highest manipulation category
+        let maxScore = 0;
+        
+        const categories = [
+        'gurusNegociosEmprendimiento', 'coachingDesarrolloPersonal', 'saludBienestarEsteticaEngañosos', 'inversionesFinanzasFraudulentas',
+        'marketingDigitalRedesSocialesInflados', 'relacionesSeduccionAmorGarantizado', 'productosRevolucionariosConAfirmacionesExageradas',
+        'educacionAdquisicionHabilidadesInstantaneas', 'espiritualidadMisticismoLeyDeLaAtraccionComercializados', 'estiloVidaExitosoFicticio'
+        ];
+        
+        categories.forEach(category => {
+        if (clasificacion[category] && clasificacion[category].porcentaje) {
+            maxScore = Math.max(maxScore, clasificacion[category].porcentaje);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)